### ISSUE TITLE
Invalid version specifiers in requirements.txt

### ISSUE THREAD

**Issue by hunter-moeller** [aws/amazon-redshift-python-driver#22]
## Client Operating System
Linux

## Python version
3.7.9

## Problem description
Package version syntax without delimiting commas is used in requirements.txt.  For example [here](https://github.com/aws/amazon-redshift-python-driver/blob/master/requirements.txt#L3).  I would expect that to be `beautifulsoup4>=4.7.0,<4.8.0`.

When installing `redshift_connector` via pip (within a conda environment), I am seeing the following error:
```
InvalidVersionSpec: Invalid version '4.7.0<4.8.0': invalid character(s)
```
which can be traced back to `beautifulsoup4>=4.7.0<4.8.0` in the requirements.txt file.

Based on [PEP 440](https://www.python.org/dev/peps/pep-0440/#version-specifiers), delimiting commas are a required part of the version clause.  I think this is likely nuanced as I am able to install `redshift_connector` via pip in some cases (for example on my local Windows machine) but hit this error when installing in a cloud Linux environment.  Is there a reason not to use separating commas?  Is it a supported syntax that I am not aware of?

**Comment by hunter-moeller** [comment]
I need to correct the above.  I am running in an error when _exporting_ a conda environment that has `redshift_connector` installed.  I am able to create the conda environment and pip install `redshift_connector` into it without issue.
```
conda env export -n my-env
```
gives
```
InvalidVersionSpec: Invalid version '4.7.0<4.8.0': invalid character(s)
```

**Comment by Brooke-white** [comment]
Hey @hunter-moeller ,

Thank you for raising this issue. The team will take a look into reproducing and resolving this issue and provide an update shortly.

**Comment by Brooke-white** [comment]
Hey @hunter-moeller ,

When taking the steps you described:
1. `conda create --name myenv`
2. `conda activate myenv`
3. `conda install pip`
4. `pip install redshift_connector`
5. `conda env export -n myenv`

I was able to reproduce this error. We will be including a fix for this in our next release of `redshift_connector`, which is scheduled to occur next week.

As a sidenote -- if `redshift_connector` is installed via conda, the environment can be exported without issue:
1. `conda create --name myenv1 redshift_connector`
2. `conda env export -n myenv1`


note: this was only tested using `redshift_connector-2.0.874`

Thanks again for bringing this to our attention! :)

**Comment by hunter-moeller** [comment]
Awesome, thank you for investigating this.

**Comment by Brooke-white** [comment]
Fix for this issue provided in [v2.0.875](https://github.com/aws/amazon-redshift-python-driver/blob/master/CHANGELOG.md#v20875-2021-03-08)
